packages/components/src/components/checkbox-group/checkbox-group.stories.ts

```diff
@@ -40,102 +43,61 @@ export const Default = {
 };
 
 /**
- * The sd-checkbox in all possible combinations of `orientation` and `size`.
+ * Use the `size` attribute to change the size of all checkboxes in the group.
```

Comment:
Use the ”size” attribute to change the size.

I would shorten it cause it changes only the size of the fonts including optional label and not the checkbox itself.

Reply:
Updated